framework/base/Model.php

```diff
@@ -773,7 +773,7 @@ public function activeAttributes()
         if (!isset($scenarios[$scenario])) {
             return [];
         }
-        $attributes = $scenarios[$scenario];
+        $attributes = array_unique($scenarios[$scenario]);
         foreach ($attributes as $i => $attribute) {
```

Comment:
Or like this already introduced in [2.0.12 Validator](https://github.com/yiisoft/yii2/blob/master/framework/validators/Validator.php#L461).